WordPress/twentytwentyfour
# Experiment: Featured image fallback

**Description**
This PR filters the `<img>` tag of the post featured image block when the post or page does not have a featured image.
It adds a 1px x 1px white (#FFFFFF) png file as the image source.
It adds the title of the post as the alt attribute.

**Idea:** Try a transparent image.

**Screenshots**

Home, default style variation:
![Featured image fallback on the home template](https://github.com/WordPress/twentytwentyfour/assets/7422055/1ed4ad8f-664c-485b-8305-ade142d12821)

Single post:
![Featured image fallback on the single post](https://github.com/WordPress/twentytwentyfour/assets/7422055/4ee7e178-effe-4fb7-aec9-8a632c941e31)

Archive:
![Featured image fallback on single post](https://github.com/WordPress/twentytwentyfour/assets/7422055/753dc727-0568-4a92-8e4c-1db47b80414b)

Home, Onyx:
![Featured image fallback on the Home of the Onyx style variation](https://github.com/WordPress/twentytwentyfour/assets/7422055/f7b8e619-9109-4f67-bde0-bc61829b5b22)

**Portfolio** home template, onyx style.
This time, with more content, using the theme test data import.
![portfolio home, onyx](https://github.com/WordPress/twentytwentyfour/assets/7422055/9e130cdc-08e4-4d21-8cf4-d9baf213584f)

**Portfolio** home template, milky style, mobile (cut off because the image was too large)
![portfolio home, milky](https://github.com/WordPress/twentytwentyfour/assets/7422055/129e6114-808c-46ea-baaf-3720a0c5ac54)


**Testing Instructions**
Make sure that your WordPress installation has posts with and without featured images.
View different content that shows featured images:
The home templates, the portfolio templates, single posts with featured images, etc.
Try different browser widths and style variations.

**Contributors**
greenshady justintadlock
